packages/contract-addresses/src/index.ts

```diff
@@ -6,6 +6,7 @@ export interface ContractAddresses {
     zrxToken: string;
     etherToken: string;
     exchange: string;
+    exchange_v3: string;
```

Comment:
Convention in TypeScript is to use `camelCase` instead of `under_scores`. Maybe we could call this `exchangeV3`?

Reply:
Sure, I'd be in favor of removing it.

Reply:
I think we should leave the old exchange address as `exchangeV2`. We'll still need it for mainnet deployments, and v2 will probably continue to be live for ~3 months after v3 is deployed.